features/admin.server-configurations.v1/forms/username-recovery-form.tsx

```diff
@@ -100,34 +107,92 @@ export const UsernameRecoveryConfigurationForm: FunctionComponent<UsernameRecove
      * @param values - Form values.
      * @returns Sanitized form values.
      */
-    const getUpdatedConfigurations = (value: boolean) => {
-        const data: { [ key: string ]: unknown } = {
-            [ GovernanceConnectorUtils.decodeConnectorPropertyName(
-                ServerConfigurationsConstants.USERNAME_RECOVERY_ENABLE) ]: value.toString()
+    const getUpdatedConfigurations = (values: Record<string, any>) => {
```

Comment:
Can we use `boolean` instead of `any` here?
```suggestion
    const getUpdatedConfigurations = (values: Record<string, boolean>) => {
```

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7004/commits/a81d0a35e1e7b077862d1d44dfde06a60da22794